v03_pipeline/lib/tasks/base/base_variant_annotations_table_test.py

```diff
@@ -49,7 +50,12 @@ def setUp(self) -> None:
             ),
         )
 
-    def test_should_create_initialized_table(self) -> None:
+    @mock.patch(
+        'v03_pipeline.lib.tasks.reference_data.updated_reference_dataset_collection.get_datasets_to_update',
+    )
+    def test_should_create_initialized_table(self, mock_get_datasets_to_update) -> None:
+        mock_get_datasets_to_update.return_value = []
```

Comment:
Yep this looks right to me!  What does this method return without the mock though!

Reply:
forgot to respond to this - if I don't mock the task it tries to read the dataset ht tables which are not mocked out and errors trying to read a file from google